shared/src/helpers/string-case/string-case.ts

```diff
@@ -0,0 +1,20 @@
+import { paramCase, Options } from 'change-case';
```

Comment:
move to `shared/src/helpers/string/change-string-case/change-string-case.helper.ts`